packages/Alert/src/index.tsx

```diff
@@ -74,34 +109,36 @@ const AlertComponent = forwardRef<'div', AlertProps>(
           alignSelf="flex-start"
           icon={icon}
           pr="md"
-          pt={1}
-          size="sm"
+          size={size}
           variant={defaultVariantIcon}
         />
-        <Box flex="1">
-          {cta ? (
-            <Box
-              alignItems={{ md: 'center' }}
-              display="flex"
-              flexDirection={{ _: 'column', md: 'row' }}
-              gap="md"
-              justifyContent="space-between"
-            >
-              <div>{content}</div>
-              {cta}
-            </Box>
-          ) : (
-            content
-          )}
+        <Box flex={1}>
+          <Box display="flex" gap="md" justifyContent="space-between" {...layout[ctaPosition]}>
+            <Box flex={1}>{content}</Box>
+
+            {!!actions && (
+              <Box alignItems="center" display="flex" gap="sm">
+                {actions}
+              </Box>
+            )}
+          </Box>
         </Box>
       </S.Alert>
     )
   }
 )
 
 // We need this component to check its existence in <Alert> and to allow users to add Button in <Alert> content
-const AlertButton = forwardRef<'button', ButtonProps>((props, ref) => (
-  <Button flexShrink={0} ref={ref} size="sm" variant="secondary" w="fit-content" {...props} />
-))
+const AlertPrimaryAction = forwardRef<'button', Omit<ButtonProps, 'size' | 'variant'>>(
+  (props, ref) => <Button flexShrink={0} ref={ref} w="fit-content" {...props} variant="secondary" />
+)
+
+const AlertSecondaryAction = forwardRef<'button', Omit<ButtonProps, 'size' | 'variant'>>(
+  (props, ref) => <Button flexShrink={0} ref={ref} w="fit-content" {...props} variant="tertiary" />
+)
 
-export const Alert = Object.assign(AlertComponent, { Title, Button: AlertButton })
+export const Alert = Object.assign(AlertComponent, {
+  Title,
+  PrimaryAction: AlertPrimaryAction,
+  SecondaryAction: AlertSecondaryAction,
```

Comment:
I think this is a breaking change, maybe we can for now export `Button: AlertPrimaryAction` ? With a deprecated message ?

Reply:
Or rename` Button / SecondaryButton` ?

Reply:
Nice catch! Yes, I tried to reduce the breaking changes on this component, but I forgot about that. You are absolutely right. I kept the current button and put the second one (see b3e2450). So, maybe we don't need deprecated anymore, it's just a styling update?